readme.md

```diff
@@ -1,3 +1,5 @@
 ### People should write more readmes
 
 Accepting pull requests
+
+![not enough gifs](https://guides.github.com/activities/hello-world/pr-tab.gif)
```

Comment:
I'm not sure this alt-text makes sense. I want to be sure that my content is accessible  to the blind and visually impaired: could you change this to a description of the gif in question?